rom/dev/src/wdt.rs

```diff
@@ -35,17 +35,12 @@ pub fn start_wdt(soc_ifc: &mut SocIfc) {
         if wdt_timeout_cycles == 0 {
             wdt_timeout_cycles = 1;
         }
-        cprintln!(
-            "[state] Starting the Watchdog Timer {} cycles",
-            wdt_timeout_cycles
-        );
+        cprintln!("[state] WDT start {} cycles", wdt_timeout_cycles);
```

Comment:
These logs were made more concise to squeeze ROM back into it's size limit with the addition of the new KAT.